doc/configcommands.dsv

```diff
@@ -8,7 +8,7 @@ bind-key||<key> <operation> [<dialog>]||n/a||Bind key <key> to <operation>. This
 bookmark-autopilot||[yes/no]||no||If set to `yes`, the configured bookmark command is executed without any further input asked from user, unless the url or the title cannot be found/guessed.||bookmark-autopilot yes
 bookmark-cmd||<command>||""||If set, then <command> will be used as bookmarking plugin. See the documentation on bookmarking for further information.||bookmark-cmd "~/bin/delicious-bookmark.sh"
 bookmark-interactive||[yes/no]||no||If set to `yes`, then the configured bookmark command is an interactive program.||bookmark-interactive yes
-browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If BROWSER environment variable is set, it will be used as the default browser, otherwise lynx will be used. Any occurrences of `%u` and `%U` in <command> will be replaced by the article's URL or the feed's URL, respectively, enclosed in single quotes.||browser "w3m %u"
+browser||<command>||%BROWSER, otherwise lynx||Set the browser command to use when opening an article in the browser. If BROWSER environment variable is set, it will be used as the default browser, otherwise lynx will be used. Any occurrences of `%u` and `%U` in <command> will be replaced according to the <<browser-format-table,browser format identifiers>>. If neither `%u` nor `%U` are included, newsboat will behave as though " `%u`" had been implicitly appended to <command>.||browser "w3m %u"
```

Comment:
> newsboat will behave as though " `%u`" had been implicitly appended to <command>

This should use single quotes, not double quotes.

I also believe that both `%u` and `%U` are always enclosed in single quotes; that was mentioned in the previous version of the doc, but is missing here.